apps/cli/src/api/up/daemon.ts

```diff
@@ -1,9 +1,15 @@
 import { Daemon } from "@opentrader/daemon";
+import { getSettings } from "../../utils/settings.js";
+
+const settings = getSettings();
+const port = settings.port;
+const host = settings.host;
 
 const daemon = await Daemon.create({
   server: {
     frontendDistPath: "../frontend",
-    port: 8000,
+    port: port,
+    domain: host,
```

Comment:
```suggestion
    host,
```

Let's maintain consistency in property naming.